docs/en/compute/Google-Dataproc.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: global
+title: Running Alluxio on Google Cloud Dataproc
+nickname: Google Dataproc
+group: Data Applications
+priority: 0
+---
+
+This guide describes how to configure Alluxio to run on [Google Cloud Dataproc](https://cloud.google.com/dataproc).
+
+* Table of Contents
+{:toc}
+
+## Overview
+
+Google Cloud Dataproc is a managed on-demand service to run Spark and Hadoop compute workloads.
+It manages the deployment of various Hadoop Services and allows for hooks into these services for
+customizations.
+Aside from the added performance benefits of caching, Alluxio also enables users to run compute 
+workloads against on-premise storage or even a different cloud provider's storage i.e. AWS, Azure
+Blob Store.
+
+## Prerequisites
+
+* Account with Cloud Dataproc API enabled
+* A GCS Bucket
+* gcloud CLI: Make sure that the AWS CLI is set up and ready with the required AWS Access/Secret key
+
+An GCS bucket can be configured as Alluxio's Root Under File System and to serve as the location for the
```

Comment:
"A GCS bucket" vs "An GCS Bucket". Also replace "can be configured" with "is required"